platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,10 +1,19 @@
 #import "MGLSource.h"
+#import "MGLTypes.h"
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface MGLRasterSource : MGLSource
 
 @property (nonatomic, readonly, copy) NSURL *URL;
 @property (nonatomic, readonly, assign) CGFloat tileSize;
+@property (nonatomic, readonly, copy) NS_ARRAY_OF(NSString *) *tileURLTemplates;
+@property (nonatomic, readonly) uint8_t minimumZoomLevel;
```

Comment:
These should be `NSInteger`-typed. In general, we should prefer Foundation typealiases over the compiler’s built-in numeric primitives in Objective-C code. It’s fine if the whole range of `NSInteger` won’t work; if it fails hard, we can clamp the value to whatever range is supported.